src/borg/archive.py

```diff
@@ -1289,6 +1292,13 @@ def verify_file_chunks(item):
                 logger.info('{}: Completely healed previously damaged file!'.format(item.path))
                 del item.chunks_healthy
             item.chunks = chunk_list
+            if 'size' in item:
+                item_size = item.size
+                item_chunks_size = item.get_size(compressed=False, from_chunks=True)
+                if item_size != item_chunks_size:
+                    logger.warning('{}: size inconsistency detected: size {}, chunks size {}'.format(
+                                   item.path, item_size, item_chunks_size))
+                    # ok, what now? we don't know whether item_size or item_chunks_size is correct.
```

Comment:
Yeah but if you'd mount or extract it you'd only get item_chunks_size bytes, no?